daemon/logger/jsonfilelog/read.go

```diff
@@ -163,7 +166,7 @@ func watchFile(name string) (filenotify.FileWatcher, error) {
 	return fileWatcher, nil
 }
 
-func followLogs(f *os.File, logWatcher *logger.LogWatcher, notifyRotate chan interface{}, since time.Time) {
+func followLogs(f *os.File, logWatcher *logger.LogWatcher, notifyRotate chan interface{}, since, until time.Time) {
```

Comment:
This function already has a lot of arguments; we can pass it `logger.ReadConfig` here